benchmarking/project.py

```diff
@@ -44,24 +44,27 @@
 @click.option(
     '-p',
     '--poisson',
-    default=(False, 0),
+    default=0,
     help='Whether to add poisson noise, and how much.',
 )
 @click.option(
     '-g',
     '--guassian',
-    nargs=3,
-    default=(False, 0, 0),
-    help='Whether to add gaussian distortions,'
-    'as well as the mean and standard deviation as well.',
+    nargs=2,
+    default=(0, 0),
+    help='Whether to add gaussian distortions.'
+    'The first entry is the mean of the guassian distribution,'
+    'the second entry is the standard deviation.',
 )
 @click.option(
     '-s_p',
     '--salt_pepper',
```

Comment:
```suggestion
    '-s',
    '--salt-pepper',
```
Generally, flags are exactly one letter or whole words separated by `-`.